tfrecorder/client.py

```diff
@@ -115,7 +116,8 @@ def _read_image_directory(image_dir: str) -> pd.DataFrame:
         row = [split, image_uri, label]
         rows.append(row)
 
-  return pd.DataFrame(rows, columns=input_schema.image_csv_schema_map.keys())
+  return pd.DataFrame(
+      rows, columns=input_schema.ImageCsvSchema.input_schema_map.keys())
```

Comment:
`Schema` should probably just subclass from `OrderedDict` or define fields() method.

Reply:
Created a getter for this.